src/diffpy/utils/tools.py

```diff
@@ -114,6 +115,11 @@ def get_user_info(args=None):
     global_config = load_config(Path().home() / "diffpyconfig.json")
     local_config = load_config(Path().cwd() / "diffpyconfig.json")
     if global_config is None and local_config is None:
+        warnings.warn(
```

Comment:
This is not quite what I had in mind.  Just the warning messages in the prompts could be clearer.  We could put this preamble message here if we really like it, but I would like to see it in the context of the full workflow pasted in the comments before deciding.  

@alisnwu have you run through the whole workflow to see what it looks like and where it can be improved?

Even if we do want this wording, I think this is not a good use of `warnings.warn`.  We are not using any of its features. A simple print statement would be clearer.

Reply:
yes, I like that (modulo a slight grammatical boo boo)!

How about something a bit more human friendly:
"No global configuration file was found containing information about the user to associate with the data.  By following the prompts below you can add your name and email to this file on the current computer and your name will be automatically associated with subsequent diffpy data by default.  This is not recommended on a shared or public computer.  You will only have to do that once.  For more information <as before>"

we need a workflow for people to update this global config file later if they want.  Pease could you make an issue to check/work on that and maybe do it on a separate PR?  Thanks!